<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVmDialog</class>
 <widget class="QDialog" name="NewVmDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New VM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="_nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="_architectureComboBox"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QRadioButton" name="_remoteTerminalRadioButton">
     <property name="text">
      <string>Remote terminal</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_architectureLabel">
     <property name="text">
      <string>Architecture:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_templateLabel">
     <property name="text">
      <string>Template:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_locationLabel">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="_browseButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="_nameLineEdit"/>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="_templateComboBox"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="_locationLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="_virtualMachineRadioButton">
     <property name="text">
      <string>Virtual machine</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="_verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_architectureComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_architectureComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_templateComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_templateComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_nameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_nameLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_locationLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_locationLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_browseButton</sender>
   <signal>clicked()</signal>
   <receiver>NewVmDialog</receiver>
   <slot>_browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_architectureComboBoxCurrentIndexChanged(int)</slot>
  <slot>_templateComboBoxCurrentIndexChanged(int)</slot>
  <slot>_accept()</slot>
  <slot>_reject()</slot>
  <slot>_nameLineEditTextChanged(QString)</slot>
  <slot>_locationLineEditTextChanged(QString)</slot>
  <slot>_browse()</slot>
 </slots>
</ui>
